dashboard/db/migrate/20231026235831_create_potential_teachers.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePotentialTeachers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :potential_teachers do |t|
+      t.string :name
+      t.string :email
+      t.string :tutorial_source
```

Comment:
what is the tutorial_source value going to be matched with exactly? for example, does it contain a code from the cdo_tutorials table, or a course_offering name, or something else?

Reply:
great! what do you think about referring to those by id, e.g. `source_course_offering_id`? using id instead of name will make it easier to do rails things, such as pull in other info from the course offering. This would also be the standard rails way of implementing a reference between objects (as long as serializing/seeding is not involved).

Reply:
yes, exactly! I think you mentioned this is your first DB migration. heads up that making changes to migrations can be tricky. if you have already run the migration locally, one option is to do `rake db:rollback` to revert the migration, then make the changes, then run the migration again. if instead you make the changes before doing the rollback, you can get into a bad state.